spec/components/schemas/content/optin_request.ts

```diff
@@ -0,0 +1,62 @@
+import { SchemaObject } from 'openapi3-ts';
+import { ref as baseRef } from './base';
+import { createComponentRef } from '../../../../utils/ref';
+
+const notificationOptIn: SchemaObject = {
+  type: 'object',
+  allOf: [{
+    $ref: baseRef,
+  }, {
+    type: 'object',
+    properties: {
+      type: {
+        type: 'string',
+        example: 'optin_request',
+      },
+      medialUrl: {
+        description: 'URL for the thumbnail the user will see.',
+        type: 'string',
+        example: 'https://cdn.zenvia.com/main-picture.jpg',
+      },
+      title: {
+        description: 'The topic of the recurring notification subscription.',
+        type: 'string',
+        example: 'Weekly Sale'
+      },
+      frequency: {
+        description: 'How often the messages will be sent after user opted-in.',
+        type: 'string',
+        enum: [
+          'DAILY',
+          'WEEKLY',
+          'MONTHLY'
+        ]
+      },
+      reOptIn: {
+        description: 'Allow user to opt-in back again after opting-out.',
```

Comment:
Na mesma linha do que falei no anterior, que tal:
`Allows the contact to opt-in back again after opting-out.`